src/tests/src/libs/antares/study/scenario-builder/test-sc-builder-file-save.cpp

```diff
@@ -176,6 +176,11 @@ struct commonFixture
         study->bindingConstraints.add("BC_2")->group("group2");
         study->bindingConstraints.add("BC_2")->group("group3");
 
+        study->bindingConstraintsGroups.add("group1");
```

Comment:
As described previously: we should not need to do this, it should be handled at the study class level when we add constraints.

Reply:
I agree.
The thing is that "adding" a bc to the study is not a use case per se. The use case is to construct a batch of bc from a study (on disk).
"Add" methods are probably useful for legacy UI but for the solver they are just useful (dangerous) helpers